docs/proposals/object-search.md

```diff
@@ -0,0 +1,160 @@
+- [Object Search Service](#object-search-service)
+  - [Motivation](#motivation)
+    - [Goals](#goals)
+  - [Proposal](#proposal)
+    - [Use Cases](#use-cases)
+  - [Design Details](#design-details)
+    - [CRD API Group and Version](#crd-api-group-and-version)
+    - [Object search service CRD](#object-search-service-crd)
+    - [Object search service type definition](#object-search-service-type-definition)
+      - [Validation](#validation)
+    - [Object search service sample](#object-search-service-sample)
+  - [Controller Design](#controller-design)
+    - [Object search service Controller](#object-search-service-controller)
+    - [Downstream Controller](#downstream-controller)
+    - [Upstream Controller](#upstream-controller)
+    - [Details of api between GM(cloud) and LC(edge)](#details-of-api-between-gmcloud-and-lcedge)
+    - [Flow of object search service creation](#flow-of-object-search-service-creation)
+  - [Workers Communication](#workers-communication)
+# Object Search Service
+## Motivation
+Object search is an important technology in the field of computer vision, which is widely used in security monitoring, intelligent transportation and other fields. Generally, object search applications have high latency requirements, which cannot be met by traditional cloud computing schemes. The target search schemes based on edge computing architecture have the characteristics of low latency and data privacy security, and are the mainstream technology trend in the industry. However, how to efficiently use the distributed resources of edge computing to improve throughput and accuracy, and reduce time delay is the current technical difficulty. We propose the first open source end-to-end multi-edge collaborative target search application implementation solution. Based on KubeEdge's cloud-edge collaboration and resource management capabilities, we use multiple edge computing nodes to execute the AI inference tasks of target search workflow in parallel to solve the above problems.
```

Comment:
we should indicate that task for large scalable may be inefficient, we can accelerate that, and for off-line search, maybe our feature can provide an efficient method to complete a search in the future.

Reply:
I revised the Motivation and added relavent description.